recipes/hsdfinder/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "HSDFinder" %}
+{% set version = "1.0.0" %}
+{% set sha256 = "f366772bdda003ca6421d39d1163c5cf2c0b3a49cde87c95193afb579feda17d" %}
```

Comment:
```suggestion
{% set sha256 = "d104c5365189115cd5864d53a3be37c02432cb7bb5e786ecd66fd835c63ad2d9" %}
```

Reply:
I see, this is great lesson for anyone else who has the similar issue, I forgot to synchronize the sha256 after modifying the name.